### ISSUE TITLE
Server-Based Validation System to Enhance Free KYC Attempts Check

### ISSUE THREAD

**Issue by bgodlin** [sora-xor/sora-card#76]

**Comment by Tieumsan** [comment]
@bgodlin what is it about ? duplicate from #75 or subtask ?

**Comment by bgodlin** [comment]
> @bgodlin what is it about ? duplicate from #75 or subtask ?

No, it's not a duplicate. Task #75 implies to removing hard-coded elements from the clients, whereas this task focuses on shifting from client-side validation to server-side validation (even though task #75 may still leave the clients with hard-coded values, they will be responsible for validation). In any case, I view this task as having a lower priority.
